WooCommerce/Classes/ViewRelated/Dashboard/DashboardView.swift

```diff
@@ -187,6 +187,9 @@ struct DashboardView: View {
         .sheet(isPresented: $viewModel.showingInAppFeedbackSurvey) {
             Survey(source: .inAppFeedback)
         }
+        .sheet(isPresented: $viewModel.showingTapToPayAwarenessMoment) {
+            TapToPayAwarenessMomentView()
+        }
```

Comment:
I don't think we use it often, but there's also a remotely controlled modal [JITM that could be shown when loading the dashboard view](https://github.com/woocommerce/woocommerce-ios/blob/11b514ac8b1ff179ca28fd32fbc3da1702517a4b/WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewHostingController.swift#L99-L114) in the UIKit way. Might be worth testing when both are presented by hard-coding the JITM modal.

Reply:
Thanks. Yes, I'm aware of JITM. I think we can disable TTP JITM remotely later for the version that this modal gets released. 

Optionally, we could consider keeping them both. Modal is quite intrusive and can get dismissed quickly while JITM just appears on the dashboard. 

I tested, this is how having both would look:

https://github.com/user-attachments/assets/c4debcea-2e41-475b-b759-59f14a582ee3